tests/component.js

```diff
@@ -0,0 +1,351 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Vue from 'vue';
+import { mount } from '@vue/test-utils';
+import CKEditorComponent from '../src/ckeditor';
+import sinon from 'sinon';
+import { getEditorNamespace } from '../src/utils/geteditornamespace';
+
+/* global window */
+const CKEDITOR = window.CKEDITOR;
+
+describe( 'CKEditor Component', () => {
+	const spies = {};
+
+	let skipReady = false;
+	let sandbox, wrapper, component, props;
+
+	beforeEach( done => {
+		spies.replace = sinon.spy( CKEDITOR, 'replace' );
+		spies.inline = sinon.spy( CKEDITOR, 'inline' );
```

Comment:
Why aren't these two spies inside the sandbox? I do not also see the need to spy on these two methods in every test.

Reply:
Good point about the sandbox :+1: However, it's just simpler to spy on them with topmost setup function, so we don't duplicate these two lines. Also, creates nice testing API with sandbox.